NEWS.md

```diff
@@ -236,8 +236,12 @@ This section lists changes that do not have deprecation warnings.
 Library improvements
 --------------------
 
+  * Functions `first` and `last` now accept `nchar` argument for `AbstractString`.
+    If this argument is used they reutrn a substring consisting of first/last `nchar`
```

Comment:
"substring" could be confused with `SubString`.

Reply:
replaced "substring" with "string"